projects/lib/src/oauth-service.ts

```diff
@@ -2305,8 +2326,21 @@ export class OAuthService extends AuthConfig implements OnDestroy {
 
         const verifier = await this.createNonce();
         const challengeRaw = await this.crypto.calcHash(verifier, 'sha-256');
-        const challange = base64UrlEncode(challengeRaw);
-
-        return [challange, verifier];
+        const challenge = base64UrlEncode(challengeRaw);
```

Comment:
I have only corrected to `const` name, not the method itself, since it is protected and I didn't want to introduce a breaking change.